src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/StartTLSCommand.scala

```diff
@@ -0,0 +1,11 @@
+package com.linagora.gatling.imap.protocol.command
+
+import com.yahoo.imapnio.async.request.{AbstractNoArgsCommand, ImapCommandType}
+
+case object StartTLSCommandType extends ImapCommandType {
+  override def getType: String = "STARTTLS"
+}
+
+class StartTLSCommand extends AbstractNoArgsCommand(StartTLSCommandType.getType) {
+  override def getCommandType: ImapCommandType = StartTLSCommandType
```

Comment:
Idem there is a dedicated method for starttls: sending the command is not enough you also need to modify the incoming pipeline to decrypt/encrypt SSL.

Reply:
I checked and there is no dedicated starttls method in imapnio. I also give a quick try modifying the SSL context in `ImapSession` but did not succeed. I will give it a try later.

Reply:
You are right, it seems not implemented.

I'm not sure StartTLS is very valuable:

 - Not part of our target deployments (that uses implicit TLS)
 - And hard to implement as it is not supported by IMAP nio
 
 I would forget about it for now.